opm/output/eclipse/RegionCache.hpp

```diff
@@ -29,22 +30,36 @@ namespace Opm {
     class Schedule;
     class EclipseGrid;
     class FieldPropsManager;
+} // namespace Opm
 
-namespace out {
+namespace Opm { namespace out {
     class RegionCache {
     public:
         RegionCache() = default;
-        RegionCache(const std::set<std::string>& fip_regions, const FieldPropsManager& fp, const EclipseGrid& grid, const Schedule& schedule);
-        const std::vector<std::pair<std::string,size_t>>& connections( const std::string& region_name, int region_id ) const;
+        RegionCache(const std::set<std::string>& fip_regions,
+                    const FieldPropsManager&     fp,
+                    const EclipseGrid&           grid,
+                    const Schedule&              schedule);
 
-        // A well is assigned to the region_id where the first connection is
+        void buildCache(const std::set<std::string>& fip_regions,
+                        const FieldPropsManager&     fp,
+                        const EclipseGrid&           grid,
+                        const Schedule&              schedule);
+
+        const std::vector<std::pair<std::string, std::size_t>>&
```

Comment:
maybe a name for the pair would make things more readable but this doesn't really leak into external user interfaces so it's probably a non-issue.

Reply:
> maybe a name for the pair would make things more readable but this doesn't really leak into external user interfaces so it's probably a non-issue.

I've added a couple of type aliases in the latest update.  Even if these don't escape to the surrounding code, they are still useful as guides for the future maintainer.